tc/sgx/trusted_worker_manager/enclave/CMakeWorkloads.txt

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##################################################################
+MACRO(CREATE_SUPPORTED_WORKLOADS_LIST)
+    SET(SUPPORTED_WORKLOADS_LIST "")
+    LIST(APPEND SUPPORTED_WORKLOADS_LIST "echo-result")
+    LIST(APPEND SUPPORTED_WORKLOADS_LIST "heart-disease-eval")
+    LIST(APPEND SUPPORTED_WORKLOADS_LIST "inside-out-eval")
+    LIST(APPEND SUPPORTED_WORKLOADS_LIST "simple-wallet")
+
+    # A new workload can be added by using below syntax
+    # LIST(SUPPORTED_WORKLOADS_LIST "<workload_id>")
+ENDMACRO()
+
+##################################################################
+MACRO(CREATE_SUPPORTED_WORKLOAD_LIBRARY_LIST)
+    SET(SUPPORTED_WORKLOAD_LIBRARY_LIST "")
+    LIST(APPEND SUPPORTED_WORKLOAD_LIBRARY_LIST "echo")
+    LIST(APPEND SUPPORTED_WORKLOAD_LIBRARY_LIST "heart_disease_eval")
+    LIST(APPEND SUPPORTED_WORKLOAD_LIBRARY_LIST "inside_out_eval")
+    LIST(APPEND SUPPORTED_WORKLOAD_LIBRARY_LIST "simple_wallet")
+
+    # A new workload library can be added by using below syntax
+    # Note: Order of workload id should match the order in
+    #       supported workload library list
```

Comment:
Please rephrase as `order of workloads and library should match in SUPPORTED_WORKLOADS_LIST  and SUPPORTED_WORKLOAD_LIBRARY_LIST`

Reply:
Done